### PR TITLE
[LIBS-295] Setup testcontainers-dynamodb

### PR DESCRIPTION
**Pull request by GZLiew**

### CODE REVIEW

File: .buildkite/pr.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      - label: ':yaml: YAML Lint'
+        plugins:
+          - seek-oss/github-merged-pr#v1.1.2:
+              mode: checkout
+          - docker#v5.7.0:
+              command: ['.']
+              image: 'cytopia/yamllint:latest'
+
+      - label: ':npm: npm install'
+        commands: npm ci
```
Comment (smilin-desperado) [inline .buildkite/pr.yml]: **suggestion:** not a big one but if you used the tap-ci utils plugin this step might be quicker

File: .buildkite/pr.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
+tapendium-aliases:
+  lib-common: &lib-common
+    - seek-oss/github-merged-pr#v1.1.2:
+        mode: checkout
+    - tapendium/1password-secrets#v2.3.0:
+        file:
+          - path: .npmrc.tpl
+            out: .npmrc
+    - nienbo/cache#v2.4.15:
```
Comment (smilin-desperado) [inline .buildkite/pr.yml:9]: **suggestion:** Add the `pipeline slug override` option in the PR and merge pipelines to reuse the npm cache between both.
e.g. https://github.com/tapendium/template-email/blob/main/.buildkite/pr.yml#L22

**Review by smilin-desperado — DISMISSED** [review]

File: .buildkite/pr.yml (hunk 1 above)
Comment (GZLiew) [reply .buildkite/pr.yml]: good suggestion! gonna use that [6102fbc](https://github.com/tapendium/testcontainers-dynamodb/pull/1/commits/6102fbc84c886a5a87a76da0f1d07bf95bce874a)

File: .buildkite/pr.yml (hunk 2 above)
Comment (GZLiew) [reply .buildkite/pr.yml:9]: yes! 6102fbc

**Review by smilin-desperado — APPROVED** [review]

**Pull request merged**